src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -1981,7 +1982,10 @@ def utask_preprocess(fuzzer_name, job_type, uworker_env):
   environment.set_value('PROJECT_NAME', data_handler.get_project_name(job_type),
                         uworker_env)
   targets_count = ndb.Key(data_types.FuzzTargetsCount, job_type).get()
-  fuzz_task_input = uworker_msg_pb2.FuzzTaskInput(targets_count=targets_count)
+  fuzz_task_input = uworker_msg_pb2.FuzzTaskInput()
+  if targets_count:
```

Comment:
Is this accounting for the case where `targets_count` can be `None`?

Reply:
I didn't. PTAL again. I'm now accounting for this.